cypress/e2e/ui-tests/test-deploy-app.cy.js

```diff
@@ -2,42 +2,49 @@ describe("Test deploying app", () => {
 
     // Tests performed as an authenticated user that
     // creates and deletes objects.
-    // user: e2e_tests_contributor_tester
+    // user: e2e_tests_deploy_app_user
 
     let users
 
     before({ defaultCommandTimeout: 100000 }, () => {
```

Comment:
I saw the `config: pageLoadTimeout=100000,baseUrl=${{ env.STUDIO_URL }}` in `.github/workflows/e2e-tests.yaml`. Perhaps we could set this value as default in the cypress config instead of having it on so many locations in the tests?